content/blog/2019/05/2019-05-22-outreachy-audit-log-project.adoc

```diff
@@ -0,0 +1,33 @@
+---
+:layout: post
+:title: 'Audit Logging in Jenkins: An Outreachy Project'
+:tags:
+- community
+- logging
+- outreachy
+:author: jvz
```

Comment:
Jenkins overlaid on https://www.outreachy.org//static/outreachy-logo-open-graph-1200x1200.bda3dc7997df.png could work.

Reply:
Also if you want to promote this blog post on social media, it might be helpful to add some [description](https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post)